toolkit/components/uniffi-bindgen-gecko-js/fixtures/tests/xpcshell/test_rondpoint.js

```diff
@@ -163,33 +163,33 @@ add_task(async function() {
   await affirmEnchaine([true, false], st.toStringBoolean.bind(st));
 
   // Bytes
-  await affirmEnchaine([-128, 127], st.toStringI8.bind(rt));
```

Comment:
The pointer type checking caught this 🕺 (I'm a little confused how it worked earlier on....)